packages/merge-styles/src/concatStyleSets.ts

```diff
@@ -134,6 +134,10 @@ export function concatStyleSets(...styleSets: (IStyleSet<any> | false | null | u
 export function concatStyleSets(...styleSets: (IStyleSet<any> | false | null | undefined)[]): IConcatenatedStyleSet<any> {
   const mergedSet: IConcatenatedStyleSet<any> = {};
 
+  if (styleSets && styleSets.length === 1 && !!styleSets[0]) {
```

Comment:
What happens if you have `[ undefined, styles ]`? I think that's actually the case [here](https://github.com/OfficeDev/office-ui-fabric-react/blob/a9053bc893181d1586aeba822ba5f56e113f7a97/packages/utilities/src/customizations/customizable.tsx#L52) where the first arg may be undefined and the second defined.

Reply:
Oops, read right over the length check. I guess there is no optimization here in the case of `[ undefined, styles ]`.

Reply:
I do not know how useful it would be, but we could filter out all the `undefined` styleSets, and check to see if the length is 1, and if so, we can return that styleSet.